lib/deck.ex

```diff
@@ -5,8 +5,8 @@ defmodule Deck do
 
   import Deck.Guards, only: [is_valid_number_of_cards: 1]
 
-  # T = 10, J = Jack, Q = Queen, K = King, A = Ace
-  @ranks ~w[2 3 4 5 6 7 8 9 T J Q K A]
+  # J = Jack, Q = Queen, K = King, A = Ace
+  @ranks ~w[2 3 4 5 6 7 8 9 10 J Q K A]
```

Comment:
This change I can't accept. `T` is a much common shortcut for 10 in "text" deck representation with `c`, `d`, `h` and `s` representing the suit.

You can map Deck's values to whatever else you want in your app.